### DISCUSSION TITLE
Installation of the official Jupyter Extension for Visual Studio Code fails

### DISCUSSION THREAD

**Discussion by francoispichard** [microsoft/vscode-jupyter#11180] [Questions and Answers]
Hello,

I have been trying to install the official Jupyter Extension for Visual Studio Code (v2022.7.1102252217) for a few hours on my computer (macOS Catalina 10.15.7, VS Code Version 1.70.2), but I keep running into the same problem. Indeed, after clicking on the "Install" button, the installation seems to start, but the status switches to "Uninstalled", which means that the extension was not installed. Have you ever encountered such situations? I have been looking for hints on this GitHub repository as well as Visual Studio Code website, but I couldn't find anything similar to my issue.

Many thanks for your help,
François

**Comment by IanMatthewHuff** [comment]
@francoispichard that's an odd one. If you try the install and then run the "Toggle Developer Tools" command do you see any errors thrown in the console section of the developer tools?

**Reply by francoispichard** [reply]
Hello @IanMatthewHuff, thanks for your quick reply. I think I understood where the problem came from: I was trying to install the Jupyter Notebook extension from a workspace with a code50 GitHub codespace, which is [CS50's adaptation of Codespaces](https://code.cs50.io/). When installing the Jupyter Extension for Visual Studio Code from a "normal" workspace, the installation works very well. However, installing the Jupyter Extension in Codespaces (_cf._ button "Install in Codespaces" in the screenshot below) leads to the aforementioned error.


<img width="1032" alt="screenshot" src="https://user-images.githubusercontent.com/58308596/186470713-a85e71d1-26c7-468f-aefc-90caa27245b0.png">

**Reply by DonJayamanne** [reply]
Could be a duplicate of https://github.com/microsoft/vscode/issues/158870#event-7247782928
Please try re-starting vscode.

**Reply by francoispichard** [reply]
Hello,

I think I figured out why the Jupyter Extension was automatically uninstalled. The `.devcontainer.json` configuration file contains a field:
```
"extension-uninstaller.uninstall": [
"ms-toolsai.jupyter"
]
```
